<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>329</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="easy">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Easy(3x3)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="normal">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal(5x5)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="hard">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hard(7x7)</string>
   </property>
  </widget>
  <widget class="QLabel" name="difficulty">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Difficulty Setting</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Setting Picture</string>
   </property>
  </widget>
  <widget class="QToolButton" name="chooseFile">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>All the settings will be 
applied at the next game.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
